src/renderer/components/opds/EntryPublicationList.tsx

```diff
@@ -9,21 +9,28 @@ import * as qs from "query-string";
 import * as React from "react";
 import { connect } from "react-redux";
 import { RouteComponentProps, withRouter } from "react-router-dom";
-import { OpdsPublicationView } from "readium-desktop/common/views/opds";
-import { PublicationView } from "readium-desktop/common/views/publication";
+import { OpdsPublicationView, OpdsResultPageInfos, OpdsResultUrls } from "readium-desktop/common/views/opds";
 import { DisplayType } from "readium-desktop/renderer/components/opds/Header";
 import GridView from "readium-desktop/renderer/components/utils/GridView";
 import ListView from "readium-desktop/renderer/components/utils/ListView";
 import Loader from "readium-desktop/renderer/components/utils/Loader";
 import { RootState } from "readium-desktop/renderer/redux/states";
 
+import PageNavigation from "./PageNavigation";
+
 interface IProps extends RouteComponentProps {
-    publications: OpdsPublicationView[] | undefined;
+    publications: OpdsPublicationView[];
+    goto: (url: string, page: number) => void;
+    urls: OpdsResultUrls;
+    page: OpdsResultPageInfos;
```

Comment:
page can be undefined, see src/common/views/opds.ts

Reply:
This comment has been addressed: https://github.com/readium/readium-desktop/pull/733/commits/2fcf478637b8b6d3ef2462ae4a5739c36cad5aa7